src/zope/ramcache/tests/test_ramcache.py

```diff
@@ -27,6 +27,12 @@
 from zope.ramcache.interfaces import ICache
 from zope.ramcache.interfaces.ram import IRAMCache
 
+def _data(values):
```

Comment:
What about using named arguments instead of expecting a list with 3 values?

Like: `def _data(value, ctime, access_count):`

Reply:
I was going for the smallest possible textual change, but since you mention it, that probably works just as well.